src/extensions/internal/extensionsconfiguration.cpp

```diff
@@ -246,7 +245,30 @@ io::paths ExtensionsConfiguration::templatesPaths() const
     return paths;
 }
 
+ValCh<io::path> ExtensionsConfiguration::extensionsPath() const
+{
+    ValCh<io::path> result;
+    result.ch = m_extensionsPathChanged;
+
+    result.val = settings()->value(USER_EXTENSIONS_PATH).toString();
+    if (result.val.empty()) {
+        result.val = settings()->defaultValue(USER_EXTENSIONS_PATH).toString();
```

Comment:
You can call setDefaultValue in ExntesionsConfiguration::init and use Settings::value directly:

 ValCh<io::path> ExtensionsConfiguration::extensionsPath() const
 {
 ValCh<io::path> result;
 result.ch = m_extensionsPathChanged;
 result.val = settings()->value(USER_EXTENSIONS_PATH);
 
 return result;
 }

Reply:
thanks, fixed